Data/ByteString.hs

```diff
@@ -1679,7 +1679,11 @@ hPutNonBlocking h bs@(PS ps s l) = do
 hPutStr :: Handle -> ByteString -> IO ()
 hPutStr = hPut
 
--- | Write a ByteString to a handle, appending a newline byte
+-- | Write a ByteString to a handle, appending a newline byte.
+--
+-- This is not atomic.
```

Comment:
It would be helpful to say what is atomic, to provide a hint for users interested to achieve property. "Unlike hPutStr, this function is not atomic" or something along this.

Reply:
This line might not be needed, because the following sentence explains the behaviour more conretely.